examples/maxwell2_fdem/emi_tutorial/Response function.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
Yeah, I have quite a few notebooks that I need to move to em_examples. These are definitely on my radar, and will do soon.